server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -216,6 +216,10 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			enableReadyCheck: true,
 			maxRetriesPerRequest: redisConfig2.maxRetriesPerRequest,
 			enableOfflineQueue: redisConfig2.enableOfflineQueue,
+			retryStrategy(times) {
```

Comment:
nit: can we change `times` to `attempts` for easier readability?

Reply:
Fixed